index.html

```diff
@@ -2,13 +2,40 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Formular</title>
```

Comment:
Let's add the actual title from the Smashing magazine page.

Reply:
done